### DISCUSSION TITLE
Remove the old documentation site

### DISCUSSION THREAD

**Discussion by lewoudar** [tortoise/tortoise-orm#1184] [General]
Hi folks,
I recently discoverd that the project has a new documentation site [here](https://tortoise.github.io/).
However, there is still this old [documentation](https://tortoise-orm.readthedocs.io/en/latest/) available and I think it will confuse newcomers. It should be removed.

**Comment by long2ice** [comment]
Dup of https://github.com/tortoise/tortoise-orm/issues/1096

**Reply by lewoudar** [reply]
Oops, sorry for the inconvenience
